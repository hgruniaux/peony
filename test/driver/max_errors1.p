//# RUN: peony -fmax-errors=2 %s

% //~ ERROR: unexpected token, found '%'
% //~ ERROR: unexpected token, found '%'
% // max errors reached, no diagnostic printed for this
