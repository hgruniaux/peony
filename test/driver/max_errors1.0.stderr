<source>:3:1: error: unexpected token, found '%'
<source>:4:1: error: unexpected token, found '%'
compilation terminated due to -fmax-errors=2.
